src/evaluators/utility/PHAL_FieldFrobeniusNorm_Def.hpp

```diff
@@ -88,8 +90,8 @@ FieldFrobeniusNormBase (const Teuchos::ParameterList& p,
     TEUCHOS_TEST_FOR_EXCEPTION (!dl->isSideLayouts, Teuchos::Exceptions::InvalidParameter,
                                 "Error! The layouts structure does not appear to be that of a side set.\n");
 
-    field      = decltype(field)(fieldName, dl->node_vector);
-    field_norm = decltype(field_norm)(fieldNormName, dl->node_scalar);
+    field      = decltype(field)(fieldName, useCollapsedSidesets ? dl->node_vector_sideset : dl->node_vector);
+    field_norm = decltype(field_norm)(fieldNormName, useCollapsedSidesets ? dl->node_scalar_sideset : dl->node_scalar);
 
     dl->node_vector->dimensions(dims);
```

Comment:
@mcarlson801  I think this should be changed to `dl->node_vector_sideset` when `useCollapsedSideset` is `true`. Same in the next two if statemets

Reply:
Yeah, this does seem to be a mistake on my part. The fix will come in with my next commit.